buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijPlugin.kt

```diff
@@ -27,6 +30,21 @@ fun Project.configureIntellijPlugin() {
     tasks.withType<BuildSearchableOptionsTask> { enabled = false }
   }
 
+  val changelog = extensions.getByType(ChangelogPluginExtension::class.java)
+
+  configure<ChangelogPluginExtension> {
+    path.set("${project.projectDir}/CHANGE-NOTES.md")
+    groups.set(emptyList())
```

Comment:
What are the `groups` by default? 🤔

Reply:
(resolved? 🤔)

Reply:
Default is: ["Added", "Changed", "Deprecated", "Removed", "Fixed", "Security"]

I figured we don't need those since we didn't use anything like this in recent updates.

Reply:
Yeah, that would be too fine-grained for our needs